analyzer/src/main/kotlin/managers/utils/NodeSupport.kt

```diff
@@ -213,3 +245,46 @@ private fun getWorkspaceMatchers(definitionFile: File): List<PathMatcher> {
         FileSystems.getDefault().getPathMatcher(pattern)
     }.orEmpty()
 }
+
+private fun getPnpmWorkspaceMatchers(definitionFile: File): List<PathMatcher> {
+    data class PnpmWorkspaces(val packages: List<String>)
+
+    val pnpmWorkspaceFile = definitionFile.parentFile.resolve("pnpm-workspace.yaml")
+
+    return if (pnpmWorkspaceFile.isFile) {
+        val workspaceMatchers = pnpmWorkspaceFile.readValue<PnpmWorkspaces>().packages
```

Comment:
Is `packages` the only attribute possible? If not, I believe we should configure that unknown attributes are to be ignored in order to not run into an exception when parsing the file.